backend/src/services/webhook/webhook-service.ts

```diff
@@ -144,12 +145,15 @@ export const webhookServiceFactory = ({
       await triggerWebhookRequest(
         webhook,
         (value) => secretManagerDecryptor({ cipherTextBlob: value }).toString(),
-        getWebhookPayload("test", {
-          workspaceName: project.name,
-          workspaceId: webhook.projectId,
-          environment: webhook.environment.slug,
-          secretPath: webhook.secretPath,
-          type: webhook.type
+        getWebhookPayload({
+          type: "test" as WebhookEvents.SecretModified,
```

Comment:
should be WebhookEvents.SecretsModified instead of "test" right?

because testing it results in the following for me:
<img width="1014" alt="image" src="https://github.com/user-attachments/assets/a7ac1aa4-686d-4bf7-a37f-72238eea60ee" />